src/storage/src/storage_state.rs

```diff
@@ -1215,6 +1213,9 @@ impl StorageState {
                         mz_ore::tracing::crate_level(&log_filter.clone().into(), "librdkafka");
                 }
 
+                // Clear out the updates as we no longer forward them to anyone else to process.
+                params.dyncfg_updates = Default::default();
```

Comment:
why do we not need them forwarded to other workers via the `InternalStorageCommand::UpdateConfiguration` below? Is there any problem with leaving them populated here?

Reply:
you caught it right as I caught it! updated!